src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+@import "../utils/_mixin.scss";
+@import "../utils/_placeholders.scss";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-right: 48px;
+  &__container {
+    width: 166px;
+  }
+
+  &__image {
+    @include container(160px);
+    display: block;
+    height: 134px;
+  }
+
+  &__caption {
+    margin: 40px 0 0;
+    font-size: 12px;
+    font-family: Roboto, sans-serif;
```

Comment:
Seems that you already have this font-family in the html tag. So you don't need to write it here again.